### ISSUE TITLE
YandexMusic support?

### ISSUE THREAD

**Issue by CoCuCoH41k** [ThatGravyBoat/Craftify#73]
Are there plans to add yandex music support for this wonderful modification? or is Russia as always on the sidelines?

**Comment by QueerZizz** [comment]
Yes, please add Yandex Music support. That would be really cool. I like the idea of the Craftify mod, but I use Yandex Music, so I didn't download it.

**Comment by nmsltr** [comment]
we wait

**Comment by matv1x** [comment]
plsss add it!

**Comment by qustareunSPm** [comment]
There is API for Ya.Music https://github.com/MarshalX/yandex-music-api
Documentation: https://yandex-music.readthedocs.io/en/main/index.html#id4
You may need a translator (DeepL)

**Comment by matv1x** [comment]
https://github.com/BadKiko/YandexMusicKAPI

**Comment by matv1x** [comment]
Pleassssssssssssse add it! We need it!
